baseapp/baseapp.go

```diff
@@ -681,7 +686,7 @@ func (app *BaseApp) beginBlock(req *abci.RequestFinalizeBlock) (sdk.BeginBlock,
 
 	if app.beginBlocker != nil {
 		ctx := app.finalizeBlockState.ctx
-		if app.MigrationModuleManager.RunMigrationBeginBlock(ctx) {
+		if app.migrationModuleManager.RunMigrationBeginBlock(ctx) {
```

Comment:
Is it going to be mandatory to set this? If so, we should add a nil check and return an error if not set or skip.

Reply:
yes, it's not mandatory, added a check